x-pack/plugins/secops/server/graphql/source_status/schema.gql.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import gql from 'graphql-tag';
+
+export const sourceStatusSchema = gql`
+  "A descriptor of a field in an index"
+  type IndexField {
+    "The name of the field"
+    name: String!
+    "The type of the field's values as recognized by Kibana"
+    type: String!
+    "Whether the field's values can be efficiently searched for"
+    searchable: Boolean!
+    "Whether the field's values can be aggregated"
+    aggregatable: Boolean!
+  }
+
+  extend type SourceStatus {
+    "Whether the configured auditbeat alias exists"
+    auditbeatAliasExists: Boolean!
+    "Whether the configured alias or wildcard pattern resolve to any auditbeat indices"
+    auditbeatIndicesExist: Boolean!
```

Comment:
It looks like we're going down the route of everything is optional? So everything has to be checked for existence before being used.

I would expect for a boolean like this it would either be true or false but not true, false, (non-existent)...But I am not 100% up on GraphQL so I might be missing something.

Reply:
You are correct